README.md

```diff
@@ -135,6 +135,11 @@ Use the "Table on Contents" menu on the top-left corner to explore the list.
 - [vite-plugin-virtual-html](https://github.com/Windson1806/vite-plugin-virtual-html) - Make Vite MPA consistent with @vue/cli.
 - [vite-plugin-posthtml](https://github.com/chromeos/static-site-scaffold-modules/tree/main/modules/vite-plugin-posthtml) - Vite plugin to run [PostHTML](https://github.com/posthtml/posthtml). Currently only runs on served/built HTML files `index.html`.
 - [vite-plugin-plain-text](https://github.com/zheeeng/vite-plugin-plain-text) - A Vite plugin transforms the rule-matched file as plain text.
+- [vite-plugin-transform](https://github.com/Silksofthesoul/vite-plugin-transform) - Plugin resolve paths in cases where standard ways dont help, and replacement functionality.
+
+#### SSR
+
+- [vite-plugin-ssr](https://github.com/brillout/vite-plugin-ssr) - Like Nuxt/Next.js but with a simpler design that works with any view framework.
```

Comment:
this plugin already exists in https://github.com/vitejs/awesome-vite#libraries

Reply:
@Scrum I think I have no right to edit the information provided by other contributors. Somehow it happened that my changes are marked as having a conflict with the edits made earlier. I know how to categorize my plugin (the one and only one), but I'm not ready to think for other contributors. Therefore, I think I'd rather undo the current change, so as not to create bugs or offend someone by incorrectly categorizing their changes.